test/collection.test.ts

```diff
@@ -59,6 +59,24 @@ describe("Bundle Module (aka Collection Module)", async () => {
       .length(0, "Bob should not have any nfts");
   });
 
+  it("should return non-zero balance of owned collection tokens", async () => {
+    const token = await nftModule.mint({
+      name: "test",
+    });
+
+    try {
+      await collectionModule.createWithERC721(nftModule.address, token.id, {
+        name: "TEST NFT",
+      });
+    } catch (err) {
+      assert.fail(err);
+    }
+
+    const nfts = await bundleModule.getOwned(adminWallet.address);
+    expect(nfts).to.be.an("array").length(1);
+    assert.equal(nfts[0].ownedByAddress, 1);
```

Comment:
you might have to do `equal(nfts[0].ownedByAddress.eq(1), true)` for equality between bignumber and a number

Reply:
Thanks Jake, updated this to `assert.isTrue(nfts[0].ownedByAddress.eq(1));`